framework/src/main/resources/config.conf

```diff
@@ -8,83 +8,69 @@ storage {
   db.engine = "LEVELDB",  // deprecated for arm, because arm only support  "ROCKSDB".
   db.sync = false,
   db.directory = "database",
-  index.directory = "index",
-  transHistory.switch = "on",
-  # You can custom these 14 databases' configs:
-
-  # account, account-index, asset-issue, block, block-index,
-  # block_KDB, peers, properties, recent-block, trans,
-  # utxo, votes, witness, witness_schedule.
 
-  # Otherwise, db configs will remain default and data will be stored in
-  # the path of "output-directory" or which is set by "-d" ("--output-directory").
+  # Whether to write transaction result in transRet store
+  transHistory.switch = "on",
 
   # setting can impove leveldb performance .... start, deprecated for arm
   # node: if this will increase process fds,you may be check your ulimit if 'too many open files' error occurs
   # see https://github.com/tronprotocol/tips/blob/master/tip-343.md for detail
-  # if you find block sync has lower performance,you can try  this  settings
-  #default = {
+  # if you find block sync has lower performance, you can try this settings
+  # default = {
   #  maxOpenFiles = 100
-  #}
-  #defaultM = {
+  # }
+  # defaultM = {
   #  maxOpenFiles = 500
-  #}
-  #defaultL = {
+  # }
+  # defaultL = {
   #  maxOpenFiles = 1000
-  #}
+  # }
   # setting can impove leveldb performance .... end, deprecated for arm
 
+  # You can custom every database' configs, Otherwise, db configs will remain default and data will
```

Comment:
change to: You can customize the configuration for each database. Otherwise, the database settings will use their defaults, and data will be stored in the "output-directory" or in the directory specified by the "-d" or "--output-directory" option.